app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
+
+> What are the reasons I should care about getting more contributors for my open source project?
```

Comment:
My biggest questions as a maintainer (especially about a year ago) was around "yeah, I get that I should delegate, I get that I should make it easier for people to contribute, and dammit I'm trying, but I just can't figure out _what_ specific things I can do to make it so". So if this promises to point me at specific things that I could do that would help, I would be sold.